### PR TITLE
testing pull request to close issues

### LINKED ISSUE #11: nuevo error para cambiar
error nuevo para correfit

### LINKED ISSUE #12: incidencia con los botones
tienes que arreglar el botón

### PR DESCRIPTION
**Pull request by cristianprofile**
please  i want to  resolve #12

### CODE REVIEW

File: Base.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+import javax.persistence.OneToMany;
+import javax.persistence.Version;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+@Entity
+public class Base {
```
Comment (cristianprofile) [inline Base.java:21]: this class is perfect!!! great!!!

File: Base.java [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    public Integer getVersion() {
+        return this.version;
+    }
+
+    public void setVersion(Integer version) {
+        this.version = version;
+    }
+
+    @Override
+    public String toString() {
```
Comment (cristianprofile) [inline Base.java:56]: method tostring ok

File: Base.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        return ReflectionToStringBuilder.toString(this,
+                ToStringStyle.SHORT_PREFIX_STYLE);
+    }
+
+    public String getName() {
+        return this.name;
```
Comment (cristianprofile) [inline Base.java:62]: return value ok

File: Base.java [hunk 4]
```diff
@@ -0,0 +1,76 @@
+package com.mylab.cromero.domain;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
```
Comment (butcherless) [inline Base.java:10]: esta linea no cuadra

Comment (cristianprofile) [inline Base.java:10]: Ftggg

**Pull request merged**